### PR TITLE
[DEID-2740] add control of ocr options

### PR DESCRIPTION
**Pull request by yuya-kitani**
### What's Changed?

ocr_options handling are added.

### PR Checklist

- [x] Updated unit tests
- [x] Ran unit tests

### CODE REVIEW

File: src/privateai_client/components/request_objects.py
Comment (letmerecall) [inline src/privateai_client/components/request_objects.py:516]: IIRC, this is `paddleocr` in deid 🤔 

Do we need to set the default here though? It'll be handled by the pydantic in deid.

**Review by letmerecall — CHANGES_REQUESTED** [review]
Had a quick look, do we need to set default ocr option here??

I feel it'll override the default set by deid container.

File: src/privateai_client/components/request_objects.py
Comment (yuya-kitani) [reply src/privateai_client/components/request_objects.py:516]: Thanks Girish! in this client, it sets default value separately. Are we okay to swap to `paddleocr` here?

e.g. 
`class ProcessedText(ProcessedMarkerText, ProcessedMaskText, ProcessedSyntheticText):
    default_type = "MARKER"
    valid_types = ["MARKER", "MASK", "SYNTHETIC"]`

File: src/privateai_client/components/request_objects.py
Comment (a-guiducci) [inline src/privateai_client/components/request_objects.py:537]: Can you format this to not include the brackets?  ', '.join(self.VALID_OCR_SYSTEM)

**Review by a-guiducci — APPROVED** [review]
Minor request, otherwise looks good if test are passing 🚀 

Let's wait until the default ocr system figured out before merging though

File: src/privateai_client/components/request_objects.py
Comment (yuya-kitani) [reply src/privateai_client/components/request_objects.py]: @a-guiducci I addressed the comment and the test passed.

File: src/privateai_client/components/request_objects.py
Comment (letmerecall) [reply src/privateai_client/components/request_objects.py]: Yes, it is the defeault OCR. Reference: https://github.com/privateai/deid/blob/acb817f62fe1b0e2dfee9d9b34fe9534713fdb95/app/models/request.py#L619

@a-guiducci which docs? docs.private-ai.com?

IMO defaulting to paddle make sense, as we ship it with the container and user can start with with with least friction.

@yuya-kitani should we bring this up in dev channel/meeting?

Comment (a-guiducci) [reply src/privateai_client/components/request_objects.py]: Ya definitely agree that paddle makes more sense

**Comment by yuya-kitani** [comment]
So I updated the default to paddleocr. Can you please re-check? @a-guiducci @letmerecall

**Comment by a-guiducci** [comment]
> So I updated the default to paddleocr. Can you please re-check? @a-guiducci @letmerecall

Looks good on my end 👍

**Comment by yuya-kitani** [comment]
> > So I updated the default to paddleocr. Can you please re-check? @a-guiducci @letmerecall
> 
> Looks good on my end 👍

Hi @letmerecall Can you please check and approve if it's okay?

**Review by letmerecall — APPROVED** [review]
Looks good to me Yuya! Thanks <3

**Comment by yuya-kitani** [comment]
> Looks good to me Yuya! Thanks <3

Thanks so much for the confirmation Girish!

**Pull request merged**
